### PR TITLE
Add new ImagePlaceholderAmp psammead component

### LINKED ISSUE #8811: [PS AMP] Transparent images show the BBC blocks placeholder behind
**Describe the bug**
A transparent image still hows the BBC blocks placeholder behind which means it can be seen through the image.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.co.uk/news/uk-55808266.amp
3. Scroll down to `Doubts over UK's medium-term vaccine roll-out`
4. See the transparent image below

**Expected behaviour**
Transparent images should not show the placeholder image behind it.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [X] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves #8811

**Overall change:**
Uses new [ImagePlaceholderAmp](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image-placeholder/README.md#imageplaceholderamp) component

**Code changes:**

- Updates `ImageWithPlaceholder` unit tests
- Implements `ImagePlaceholderAmp` into`ImageWithPlaceholder` unit tests
- Adds a storybook story
- Updates snapshot tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by HarveyPeachey** [comment]
When bumping `psammead-image-placeholder` to `3.1.1` and integrating the new amp component, the PR now fails on a few existing canonical integration and e2e tests due to them not being able to find generated classnames. Those being [`div[class*='LoadingImageWrapper']`](https://github.com/bbc/simorgh/blob/29b7cc88e0ea6a1f0ee53aa4d70b44f35a2350e9/src/integration/common/mediaPlaceholder.js#L5) in the integration tests for `OnDemandTV` and `MediaAssetPage` with the asset `/pashto/bbc_pashto_tv/tv_programmes/w13xttn4` being an example, and [`div[class*="ImagePlaceholder"]`](https://github.com/bbc/simorgh/blob/29b7cc88e0ea6a1f0ee53aa4d70b44f35a2350e9/cypress/integration/pages/articles/testsForCanonicalOnly.js#L38), [`div[class*="StyledVideoContainer"]`](https://github.com/bbc/simorgh/blob/29b7cc88e0ea6a1f0ee53aa4d70b44f35a2350e9/cypress/integration/pages/articles/testsForCanonicalOnly.js#L90) article e2e tests with `/news/articles/cn7k01xp8kxo` being one of the assets tested.

The classes are basically rendering without the emotion styled suffix at the end, this can be shown by running `yarn run e2e:test:interactive` and running the article tests and visiting the page fired up by the server http://localhost:7080/news/articles/cn7k01xp8kxo. 

However when running the dev server with `yarn run dev` and visiting the assets being tested the suffix is showing in the class name. I'm not entirely sure how to proceed from here. The only thing that's really different is the bumping of the psammead packages to pull in the changes for image placeholder. The logic should be the same. It seems that the styled emotion classnames aren't appending for some reason on the bumped psammead components.

**Review by HarveyPeachey — COMMENTED** [review]
I'm a bit concerned that this has added an extra 20 seconds to run the unit tests on my local machine against the main branch. Would be nice if people could check this on their machines as well.

I think this is due to the image placeholder component adding 6 amp fallback/placeholder images for each image on the page and snapshots like `frontpage` with loads of images where each one has an extra 6 images added to it takes longer for jest to make it's comparisons. The snapshot's don't reflect the implementation, as each amp image is fetched and rendered based on [screen size](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout#element-media-queries), meaning each image will have one fallback and one placeholder amp image not 6.

What are peoples thoughts on this? We could just go for the quick and easy fix here: https://github.com/bbc/simorgh/issues/8811#issuecomment-772684278

**Comment by andrewscfc** [comment]
> I'm a bit concerned that this has added an extra 20 seconds to run the unit tests on my local machine against the main branch. Would be nice if people could check this on their machines as well.
> 
> I think this is due to the image placeholder component adding 6 amp fallback/placeholder images for each image on the page and snapshots like `frontpage` with loads of images where each one has an extra 6 images added to it takes longer for jest to make it's comparisons. The snapshot's don't reflect the implementation, as each amp image is fetched and rendered based on [screen size](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout#element-media-queries), meaning each image will have one fallback and one placeholder amp image not 6.
> 
> What are peoples thoughts on this? We could just go for the quick and easy fix here: [#8811 (comment)](https://github.com/bbc/simorgh/issues/8811#issuecomment-772684278)

Just quickly thinking about this, I'd say the problem is us using snapshots to capture whole pages for tests. The fact we have an issue with our tests shouldn't influence the fix we choose imo. I'd suggest investigating alternatives to the affected tests and very seriously considering removing some as I don't think they add a lot of value.

File: src/app/containers/ImageWithPlaceholder/index.jsx [hunk 1]
```diff
@@ -42,6 +45,10 @@ const ImageWithPlaceholder = ({
     <Image onLoad={() => setIsLoaded(true)} {...imageProps} />
   );
 
+  const AmpImgWrapper = styled.div`
+    position: relative;
```
Comment (andrewscfc) [inline src/app/containers/ImageWithPlaceholder/index.jsx:49]: what is this css needed for?

Comment (andrewscfc) [reply src/app/containers/ImageWithPlaceholder/index.jsx:49]: Raise psammead PR as discussed to move this into psammead after merge

**Review by andrewscfc — APPROVED** [review]
As discussed imo adding 20 seconds to the tests is not critical but we should follow up straightaway with a PR to remove/replace the full page snapshot tests that are affected by this change.

**Review by FK78 — APPROVED** [review]

**Comment by LilyL0u** [comment]
@HarveyPeachey In the original issue the example for this is on live. Do you have a local asset with which I can check if this has been fixed? Or a test asset to check it if we put it on test first?

**Comment by paruchurisilpa** [comment]
Tested using localhost:7080/news/uk-55808266.amp by running a live build and the transparent images do not show the BBC blocks placeholder behind. Attached is the screenshot.

LGTM..
Screenshot 2021-04-06 at 16 54 04

**Pull request merged**